source/quick-start-sinatra/download-and-install.txt

```diff
@@ -0,0 +1,111 @@
+.. _mongoid-qs-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: ruby framework, odm, code example
+
+Prerequisites
+-------------
+
+To create the Quick Start application by using Sinatra, you need the
+following software installed in your development environment:
+
+- `Ruby <https://www.ruby-lang.org/en/documentation/installation/>`__
+- `RubyGems package manager <https://rubygems.org/pages/download>`__
+- A terminal app and shell. For MacOS users, use Terminal or a similar app.
+  For Windows users, use PowerShell.
+
+Download and Install the Mongoid Gem
+------------------------------------
+
+Complete the following steps to install and add the {+odm+} gem
+to your web application.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install {+odm+}
+
+      Ensure that the version of {+odm+} you install is compatible with the
+      version of {+language+} installed on your operating system. To
+      learn which versions are compatible, see the
+      :ref:`mongoid-compatibility` page.
+
+      Run the following command to install {+odm+}:
+
+      .. code-block:: bash
+
+         gem install mongoid
+
+      In {+language+}, packages are called **gems**.
```

Comment:
S: Move this line to immediately before "Run the following command..." a few lines up